src/components/data-processing/MitochondrialContent/MitochondrialContent.jsx

```diff
@@ -102,8 +103,12 @@ const MitochondrialContent = (props) => {
 
   useEffect(() => {
     if (config && plotData && expConfig) {
-      const newConfig = _.clone(config);
-      _.merge(newConfig, expConfig);
+      let newConfig = _.clone(config);
+
+      const expConfigSettings = expConfig.methodSettings[expConfig.method];
```

Comment:
This change is to make the bin step update immediately, we can't merge expConfig with newConfig immediately as in other steps because in this one expConfig has its config stored in methodSettings